### PR TITLE
User search

### PR DESCRIPTION
**Pull request by jrossett**
User search to be able to 1. see the profile 2. click on the message icon on the profile and message them

### CODE REVIEW

**Comment by alemannosara** [comment]
I already did the part where you can see the user's profile when you click on it. I did not read the code yet but it could be fine not to have duplicate code for the same task

File: app/src/main/java/ch/epfl/sweng/thegramginers/ProfileFragment.java [hunk 1]
```diff
@@ -510,6 +502,37 @@ public void onClick(View v) {
...
+                            if (evt.getPropertyName().equals(Conversation.DATABASE_ELEMENT_FIELD_IS_SYNCHRONIZED) && newConv.isSynchronized()) {
+                                newConv.removePropertyChangeListener(this);
+                                Bundle bundle = new Bundle();
+                                bundle.putString("conversationId", newConv.getId());
+                                ConversationFragment conversationFragment = new ConversationFragment();
+                                conversationFragment.setArguments(bundle);
+                                NavigationActivity activity = (NavigationActivity) view.getContext();
+                                activity.getSupportFragmentManager().beginTransaction()
+                                        .replace(R.id.frame, conversationFragment)
+                                        .addToBackStack(null)
```
Comment (alemannosara) [inline app/src/main/java/ch/epfl/sweng/thegramginers/ProfileFragment.java]: What does "addToBackStack(null)" do?

**Comment by jrossett** [comment]
On click, it goes on the part you already did but I added a button with an message image on the profile to be able to message them.

File: app/src/main/java/ch/epfl/sweng/thegramginers/ProfileFragment.java (hunk 1 above)
Comment (jrossett) [reply app/src/main/java/ch/epfl/sweng/thegramginers/ProfileFragment.java]: The addToBackStack(null) is used in order to go back to this fragment upon clicking the 'Back' button

**Pull request merged**